README.md

```diff
@@ -149,8 +149,52 @@ git clone https://github.com/geostyler/geostyler-demo.git
 cd geostyler-demo
 npm i
 npm link geostyler
+```
+
+When working with npm link it may happen that some tools like webpack or typescript don't know how to resolve packages that are used in booth packages (in this case `geostyler` and `geostyler-style`). So we have to configure this in the `geostyler-demo`:
```

Comment:
```suggestion
When working with npm link it may happen that some tools like webpack or typescript don't know how to resolve packages that are used in both packages (in this case `geostyler` and `geostyler-style`). So we have to configure this in the `geostyler-demo`:
```